docs/SOPs/Globus_data_how_to.md

```diff
@@ -0,0 +1,109 @@
+---
+layout: default
+title: How to get data from Globus
+parent: SOPs
+has_children: false
+last_modified_date: 7/04/2022
+---
+
+# How to get data from Globus
+
+[Globus](https://www.globus.org/) is a service that makes it easy to move, sync, and share large amounts of data. Globus can manage file transfers, monitor performance, retry failures, recover from faults automatically when possible, and report the status of data transfer. Globus uses GridFTP for more reliable and high-performance file transfer, and will queue file transfers to be performed asynchronously in the background.
+Globus was developed and is maintained at the University of Chicago and is used extensively at supercomputer centres and major research facilities. 
+
+Globus can support different types of storage and present them through a unified interface to simplify file transfer. 
+A resource - server, cluster, storage system, laptop, or other system - is identified through an endpoint: an endpoint is defined, it will be available to authorised users who can transfer files to or from this endpoint.
+
+## General flow
+Using GlobusPersonalConnect you can establish a private endpoint in your machine, for example in the ec2 instance. 
+You can then transfer files from the public endpoint shared by the scientists to your machine through your the private endpoint.
+Once the data is in your filesystem you can upload it to a bucket with hca-util.
+
+To avoid transfering the files twice we could potentially mount the hca-util bucket to the ec2 and transfer the files directly in the hca-util bucket. There are guides on how to mount s3 buckets but that would require a dev because wranglers don’t have sudo powers in the ec2.
+
+#### Step 1: Sign up
+Register on globus [here](https://auth.globus.org/p/login?client_id=89ba3e72-768f-4ddb-952d-e0bb7305e2c7&scope=urn%3Aglobus%3Aauth%3Ascope%3Aauth.globus.org%3Aview_identities+urn%3Aglobus%3Aauth%3Ascope%3Anexus.api.globus.org%3Agroups+urn%3Aglobus%3Aauth%3Ascope%3Atransfer.api.globus.org%3Aall&response_type=token&redirect_uri=%2Fv2%2Foauth2%2Fauthorize%3Fclient_id%3D89ba3e72-768f-4ddb-952d-e0bb7305e2c7%26scope%3Durn%253Aglobus%253Aauth%253Ascope%253Aauth.globus.org%253Aview_identities%2520urn%253Aglobus%253Aauth%253Ascope%253Anexus.api.globus.org%253Agroups%2520urn%253Aglobus%253Aauth%253Ascope%253Atransfer.api.globus.org%253Aall%26response_type%3Dtoken%26redirect_uri%3Dhttps%253A%252F%252Fapp.globus.org%252Flogin%26redirect_name%3DGlobus%2520Web%2520App%26state%3Dewa5ysmp6xj%26viewlocale%3Den_US&redirect_name=Globus+Web+App&viewlocale=en_US)\
```

Comment:
Use this link instead as the current link is for private browsing

https://auth.globus.org/p/login?client_id=89ba3e72-768f-4ddb-952d-e0bb7305e2c7&scope=urn%3Aglobus%3Aauth%3Ascope%3Aauth.globus.org%3Aview_identities+urn%3Aglobus%3Aauth%3Ascope%3Anexus.api.globus.org%3Agroups+urn%3Aglobus%3Aauth%3Ascope%3Atransfer.api.globus.org%3Aall&response_type=token&redirect_uri=%2Fv2%2Foauth2%2Fauthorize%3Fclient_id%3D89ba3e72-768f-4ddb-952d-e0bb7305e2c7%26scope%3Durn%253Aglobus%253Aauth%253Ascope%253Aauth.globus.org%253Aview_identities%2520urn%253Aglobus%253Aauth%253Ascope%253Anexus.api.globus.org%253Agroups%2520urn%253Aglobus%253Aauth%253Ascope%253Atransfer.api.globus.org%253Aall%26response_type%3Dtoken%26redirect_uri%3Dhttps%253A%252F%252Fapp.globus.org%252Flogin%26redirect_name%3DGlobus%2520Web%2520App%26state%3Dmq2sv0x1w8h%26viewlocale%3Den_US&redirect_name=Globus+Web+App&viewlocale=en_US